toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,637 @@
+function openNav() {
+    document.getElementById("mySidenav").style.width = "250px";
+    document.getElementById("main").style.marginLeft = "250px";
+}
+
+function closeNav() {
+    document.getElementById("mySidenav").style.width = "0";
+    document.getElementById("main").style.marginLeft = "0";
+}
+
+function myreload() {
+    location.reload();
+}
+
+
+
+function insertIntoTopKTable() {
+    var table = document.getElementById("top-5-match-table");
+    var row = table.insertRow(-1);
+    var row2 = table.insertRow(-1);
+    data.topKStats.forEach(function (element) {
```

Comment:
Codacy found an issue: ['data' is not defined.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)